src/basilisp/lang/obj.py

```diff
@@ -202,6 +202,12 @@ def _lrepr_bool(o: bool, **_) -> str:
     return repr(o).lower()
 
 
+@lrepr.register(bytes)
+def _lrepr_bytes(o: bytes, **_) -> str:
+    v = repr(o)
+    return f'#b "{v[2:-1]}"'
```

Comment:
This seems to just be the easiest way to remove the surrounding `b'` and `'` that doesn't involve creating a new string twice (using `lstrip` and `rstrip`).